spec/classes/foreman_config_apache_spec.rb

```diff
@@ -15,6 +15,44 @@
         end
       end
 
+      describe 'without manage_selinux_booleans' do
+        let :facts do
+          override_facts(super(), os: {'selinux' => {'enabled' => true}})
```

Comment:
I suppose.  I think the nicer way of testing would be to make the tests conditional on `osfamily` instead.